AAI/AAIConnectProfile.md

```diff
@@ -84,55 +95,38 @@ or by providing access to the underlying resources directly (i.e. the Passport
 Clearinghouse may act like a resource server). Some Passport Clearinghouses may
 issue Passports that contain a new set or subset of Visas for downstream consumption.
 
-<a name="term-data-holder"></a> **Data Holder** -- An organization that
-holds a specific set of data (or its copy) and respects
-and enforces the data controller's decisions on who can access it. A data controller
-can also be a data holder. Data holders run a
-[Passport Clearinghouse Server](#term-passport-clearinghouse) at a minimum.
-
-<a name="term-data-controller"></a> **Data Controller** -- An organization that manages
-data and, in that role, has capacity to decide who can access it. For
-instance, a Data Access Committee (DAC).
-
 <a name="term-passport-scoped-access-token"></a> **Passport-Scoped Access Token** --
 An OIDC access token with [scope](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3)
 including the identifier `ga4gh_passport_v1`.
 
 The access token MUST be a JWS-encoded JWT token containing `openid` and `ga4gh_passport_v1`
-entries in the value of its `scope` claim.
-It is RECOMMENDED that Passport-Scoped Access Tokens follow the [RFC9068 JWT Profile for OAuth 2.0 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068) specification.
+entries in the value of its `scope` [Claim](#term-claim).
+It is RECOMMENDED that Passport-Scoped Access Tokens follow the JWT Profile for OAuth 2.0 Access Tokens specification [[RFC9068]](#ref-rfc9068).
 
-<a name="term-passport"></a> **Passport** -- A signed and verifiable JWT container for holding [Visas](#term-visa).
+<a name="term-passport"></a> **Passport** -- as defined by [[Passport]](#ref-passport) -- A signed and verifiable JWT that contains [Visas](#term-visa).
```

Comment:
This would be a definition by circle, because the GA4GH Passports spec does not define Passport, it links to this specification for its definition. I recommend to remove the first part:
```suggestion
<a name="term-passport"></a> **Passport** -- A signed and verifiable JWT that contains [Visas](#term-visa).
```